src/main/java/bowling/domain/frame/NormalFrame.java

```diff
@@ -66,4 +72,36 @@ public int getFrameNo() {
     public StateDtos getFrameResult() {
         return StateDtos.of(state.getState());
     }
+
+    @Override
+    public Score getScore() {
+        if (this.validateIsNotAddableScore()) {
+            return Score.UN_SCORE;
+        }
+
+        final Score score = state.getScore();
+        if (score.isZeroOfExtraBonusCount()) {
+            return score;
+        }
+
+        return nextFrame.calculateAdditionalScore(score);
+    }
+
+    @Override
+    public Score calculateAdditionalScore(final Score beforeScore) {
+        final Score score = state.calculateScoreForExtraBonusCount(beforeScore);
+        if (score.isZeroOfExtraBonusCount()) {
+            return score;
+        }
```

Comment:
```suggestion
        if (score.isZeroOfExtraBonusCount()) {
            return score;
        }
        final Score score = state.calculateScoreForExtraBonusCount(beforeScore);
```
위에 쓴것처럼
이런식으로 하고, 더한 점수가 계산 가능한지 아닌지는 이 다음 프레임으로 넘기는게 좋을거 같아요